src/modules/upload_scores.js

```diff
@@ -0,0 +1,26 @@
+import {
+  baseUrl, games, id, scores,
+} from './api.js';
+import recentScores from './recent_scores.js';
+import loadScores from './load_scores.js';
+
+const scoresContainer = document.querySelector('.scores-list');
+const path = `${baseUrl}${games}${id}${scores}`;
+async function uploadScore(user, score) {
```

Comment:
- [ ] Please try to prefer using the newer ES6 syntax. In this case please switch this `function` syntax` with an arrow function
```suggestion
const uploadScore = async (user, score) => {
```